client/my-sites/email/email-management/home/email-plan.jsx

```diff
@@ -254,17 +256,47 @@ class EmailPlan extends React.Component {
 		);
 	}
 
-	render() {
-		const {
-			domain,
-			selectedSite,
-			hasSubscription,
-			purchase,
-			isLoadingPurchase,
-			translate,
-		} = this.props;
+	renderRenewalNavItem() {
+		const { purchase, selectedSite, translate } = this.props;
+
+		return (
+			<VerticalNavItem>
+				<RenewButton
+					purchase={ purchase }
+					selectedSite={ selectedSite }
+					subscriptionId={ parseInt( purchase.id, 10 ) }
```

Comment:
I would suggest this shorter and clearer approach:

```suggestion
					subscriptionId={ Number( purchase.id ) }
```